client/engine/metrics.go

```diff
@@ -122,14 +122,6 @@ func (o *MetricsRecorder) RecordObjectiveCompleted(id protocols.ObjectiveId) {
 
 }
 
-// RecordOutgoingMessage records metrics about the the outgoing message
-func (o *MetricsRecorder) RecordOutgoingMessage(msg protocols.Message) {
```

Comment:
We may want to reimplement this so we're still capturing metrics about messages.

Reply:
Actually we could probably record these metrics in a message service, which may be a better place for them.